src/lib.rs

```diff
@@ -3334,6 +3334,21 @@ impl Build {
                     // Use the GNU-variant to match other Unix systems.
                     name = format!("g{}", tool).into();
                     self.cmd(&name)
+                } else if target.os == "vxworks" {
+                    name = format!("wr-{}", tool).into();
+                    self.cmd(&name)
+                } else if target.os == "nto" {
+                    name = match target.arch {
+                        "i586" | "x86" => format!("ntox86-{}", tool).into(),
+                        "aarch64" | "x86_64" => format!("nto{}-{}", target.arch, tool).into(),
+                        _ => {
+                            return Err(Error::new(
+                                ErrorKind::InvalidTarget,
+                                format!("Unknown architecture for Neutrino QNX: {}", target.arch),
+                            ))
+                        }
+                    };
+                    self.cmd(&name)
                 } else if self.get_is_cross_compile()? {
                     match self.prefix_for_target(&self.get_raw_target()?) {
                         Some(prefix) => {
```

Comment:
If you really want this to continue working in the future, it'd be nice if you could add either a test, or ideally a CI step that sets up the required environment for NTO and vxWorks and runs the normal test suite.